python/lsst/meas/mosaic/checkMosaicTask.py

```diff
@@ -424,7 +424,7 @@ def writeCatalog(self, allSource, wcsDic, calibDic, ffpDic):
         outHdu.writeto("catalog_check.fits", clobber=True)
 
 
-    def check(self, dataRefList, ct=None, debug=False, verbose=False):
+    def run(self, dataRefList, ct=None, debug=False, verbose=False):
```

Comment:
Don't you also need to update the call to `self.check(...)`, below?

Reply:
Yes.  Amended the commit to include this.